src/styles/block/card.scss

```
@@ -0,0 +1,130 @@
+.main {
+  @include display-justify-content;
+}
+
+.catalog {
+  width: 1024px;
+  @include display-justify-content;
+  flex-wrap: wrap;
+  padding: 0 40px;
+  margin: 0 auto;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  padding: 15px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid $c-border;
+  border-radius: 5px;
+  &--pos {
+    margin-bottom: 50px;
+  }
+  &__photo {
+    margin: 0 auto 42px;
+  }
+  &__photo-img {
+    box-sizing: content-box;
+    width: 160px;
+    height: 134px;
+    padding-top: 15px;
+    border: 3px solid transparent;
+  }
+  &__content {
+    margin: 0 14px;
+  }
+  &__title {
+    margin-bottom: 5px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+  }
+  &__code {
+    margin-bottom: 13px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 11px;
+    line-height: 13px;
+  }
+  &__rate {
+    display: flex;
+    height: 19px;
+    margin-bottom: 18px;
+    align-items: flex-end;
+  }
+  &__stars {
+    display: flex;
+    margin-right: auto;
+    margin-left: 1px;
+  }
+  &__star {
+    flex-basis: 21px;
+    padding-right: 6px;
+  }
+  &__rewiew {
+    position: relative;
+    bottom: 3px;
+    align-items: baseline;
+    flex-basis: auto;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 13px;
+  }
+  &__price {
+    display: flex;
+    height: 15px;
+    margin-bottom: 24px;
+    align-items: baseline;
+  }
+  &__price-info {
+    font-size: 11px;
+    flex-basis: 50%;
+  }
+  &__price-amount {
+    flex-basis: 50%;
+    display: flex;
+    margin-bottom: -1px;
+    flex-direction: row-reverse;
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+  }
+  &__buy {
+    @include display-justify-content;
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    box-sizing: border-box;
+
+    width: 170px;
+    height: 40px;
+    margin: 0 0 15px;
+    align-items: center;
+
+    font-size: 14px;
+    font-weight: 700;
```

використовуй переменні  для стилей тексту, наприклад
```suggestion
    font-weight: $fw-bold;
```